syntax = "proto3";

package example.v1;

import "buf/validate/validate.proto";

message TenantRef {
  string tenant_id = 1 [(buf.validate.field).string = {
    min_len: 1
    max_len: 120
  }];
  string region = 2 [(buf.validate.field).string = {
    min_len: 2
    max_len: 20
  }];
}
